ma/maBalance.cc

```diff
@@ -135,7 +134,7 @@ double estimateWeightedImbalance(Adapt* a)
 
 void preBalance(Adapt* a)
 {
-  if (PCU_Comm_Peers()==1)
+  if (a->mesh->getPCU()->Peers()==1)
```

Comment:
@cwsmith @jacobmerson I got rid of all the >4 dereference chains, there only remain about 10 of these 4 long chains. Let me know if you'd like me to shorten them, although I think this format can sometimes make it easier to immediately see where the mesh is coming from.

Reply:
I'm actually fine with the dereference chains. If Cameron prefers, just make a local variable with `PCU*` at the top of all of those functions.